### ISSUE TITLE
Terrorblade crashes the game

### ISSUE THREAD

**Issue by Ori-Pixel** [ValveSoftware/Dota-2#2691]
From #2689 
Different issue that the original poster's game crash bug. Appears to be from terrorblade, initially thought was a select all units hotkey bug, but seems to only be from terrorblade, doesn't matter what team of if you are the one controlling them in game.

steam runtime diagnostic gist https://gist.github.com/Ori-Pixel/1865f51422a7c506d9925413cb47dfb6

------------------------

 Having the same issue. Got an abandon after crashing about 30 times in a single match, pretty fucking awful experience. I normally got only a single crash a game until the patch around 3pm Seattle time. idk what was in that 130MB but it FUCKED my system badly.

Also, thanks to the changes from a few years ago, there are no dump files to upload. 

All I know is you can reproduce it very easily and instantly by demo mode terrorblade and have an enemy, spawn 3 illusions, click select all. Crashes every time.

Here is a speedrun video of forcing the crash:
[output.webm](https://github.com/ValveSoftware/Dota-2/assets/54956527/723c8f18-d498-40b5-8361-72862182c959)

Edit: for some reason this only works with terror lade for me? At least Naga illusions didn't trigger it.

_Originally posted by @Ori-Pixel in https://github.com/ValveSoftware/Dota-2/issues/2689#issuecomment-2067361162_

**Comment by kisak-valve** [comment]
Hello @Ori-Pixel, please copy the contents of Steam Runtime Diagnostics from Steam (`Steam` -> `Help` -> `Steam Runtime Diagnostics`) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report.

**Comment by xlanor** [comment]
I've got this as well. I crash every single time I micro illusions with TB. I havent played against TB but will be banning him every game now.

Wayland with KDE plasma 6, using vulkan to run. also on arch. 

[Runtime Diagnostics](https://gist.github.com/xlanor/0bf43b5bdfb3a07e166a2c6c6ca42266)

**Comment by Gl1tchA** [comment]
I've also experiencing this as well, running on -vulkan and using two immortals of terror with arcana

**Comment by timpostigen** [comment]
Same, running Arch linux using XFCE, game segfaults right after spawning manta illusions and group selecting.

From dmesg:
```
[timestamp] VKRenderThread[9593]: segfault at 8 ip 00007362709bdd8d sp 00007362582fb470 error 6 in librendersystemvulkan.so[7362708e1000+7f9000] likely on CPU 8 (core 4, socket 0)
[Sun Apr 21 10:17:29 2024] Code: 89 72 18 48 83 c2 14 31 c0 87 02 48 8d 95 48 ff ff ff 4c 89 ff be ff ff 00 00 e8 7e d6 ff ff 48 8b 85 48 ff ff ff 48 8b 78 60 <f0> 83 6f 08 01 75 06 48 8b 07 ff 50 08 48 8b 85 48 ff ff ff 48 c7
```

[Steam Runtime Diagnostics](https://gist.github.com/timpostigen/db75e54ccf3adfd9e00390b0cd64c1d9) gist

**Comment by mpdgraev** [comment]
Same here on manjaro linux, fully reproducible:

Selecting 2 terrorblade illusions, either via drag-selecting or a hotkey, instantly freezes the game, forcing me to kill it.

PL, naga and chaos knight do not have this issue.

[steam runtime diagnostics](https://gist.github.com/mpdgraev/386c47eaeab277768f4693d37ce7680e)

**Comment by ThinkRedstone** [comment]
I also have this exact same issue- using select all units while having two TB illusions out crashes the game

**Comment by danginsburg** [comment]
I have this reproduced, investigating.

**Comment by danginsburg** [comment]
We have root caused this and have a fix that will ship in the next Dota client update.

**Comment by timpostigen** [comment]
Wonderful, thank you!

**Comment by Ori-Pixel** [comment]
> We have root caused this and have a fix that will ship in the next Dota client update.

Can confirm fixed in the update on 23.04 ~10 pm Pacific Time
